doc/conf.py

```diff
@@ -122,13 +124,15 @@ def find_version(*file_paths):
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-on_rtd = os.environ.get("READTHEDOCS", None) == "True"
+if os.environ.get("READTHEDOCS", None) == "True":
+    html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "")
```

Comment:
I think this has to actually define the URL? https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar-READTHEDOCS_CANONICAL_URL

Reply:
(I didn’t get a notification for this :()

What do you mean exactly? My understanding is that this is set by RTD according to the external domain that is configured inside of RTD?

ie this one
<img width="843" alt="image" src="https://github.com/user-attachments/assets/dafc6ffb-cbb9-4ebd-98e1-bbf3dc2e10c4">

Reply:
If that's the case then great 😄